configure

```diff
@@ -738,6 +738,12 @@ def configure_node(o):
   if options.enable_static:
     o['variables']['node_target_type'] = 'static_library'
 
+  modules = ['src/node.js']
+  o['variables']['library_files'] = modules
+  lib_dir = os.path.join(root_dir, 'lib')
+  for root, dirs, files in os.walk(lib_dir):
+    modules += map(lambda xs: os.path.join(root, xs)[2:], files)
```

Comment:
Let us say that my editor creates a temporary file, then if I run `./configure`, it will include that as well. Probably, we need to filter out the files which ends with `.js`.